### PR TITLE
Removal of Legacy Views on 6.0.0 beta 2

### PR DESCRIPTION
**Pull request by bordoni**
## [Documentation on what will be removed](https://github.com/the-events-calendar/the-events-calendar/blob/feature/remove-legacy-views-beta-2.2/legacy-views.md)

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -3598,47 +2911,6 @@ public function save_organizer_data( $postID = null, $post = null ) {
...
 
-		/**
-		 * Add a new Organizer
-		 *
-		 * @param      $data
-		 * @param null $post
-		 *
-		 * @return int|WP_Error
-		 */
-		public function add_new_organizer( $data, $post = null ) {
```
Comment (borkweb) [inline src/Tribe/Main.php:3609]: Brought this back and deprecated it. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3043))

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -3826,76 +2990,6 @@ public function verify_unique_name( $name, $post_type ) {
...
-		/**
-		 * Given a date (YYYY-MM-DD), returns the first of the next month
-		 * hat tip to Dan Bernadict for method cleanup
-		 *
-		 * @param string $date
-		 *
-		 * @return string Next month's date
-		 * @throws OverflowException
-		 */
-		public function nextMonth( $date ) {
```
Comment (borkweb) [inline src/Tribe/Main.php:3838]: Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3139))

File: src/Tribe/Main.php [hunk 3]
```diff
@@ -3826,76 +2990,6 @@ public function verify_unique_name( $name, $post_type ) {
...
-		/**
-		 * Given a date (YYYY-MM-DD), return the first of the previous month
-		 * hat tip to Dan Bernadict for method cleanup
-		 *
-		 * @param string $date
-		 *
-		 * @return string Previous month's date
-		 * @throws OverflowException
-		 */
-		public function previousMonth( $date ) {
```
Comment (borkweb) [inline src/Tribe/Main.php:3873]: Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3178))

File: src/Tribe/Main.php [hunk 4]
```diff
@@ -3940,21 +3034,6 @@ public function eventMetaBox() {
...
 		}
 
-		/**
-		 * Get the date string (shortened).
-		 *
-		 * @param string $date The date.
-		 *
-		 * @return string The pretty (and shortened) date.
-		 */
-		public function getDateStringShortened( $date ) {
```
Comment (borkweb) [inline src/Tribe/Main.php:3950]: Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3259))

**Review by borkweb — COMMENTED** [review]
I've made it through all files leading up to `src/Tribe/Main`. Taking a break and will come back to it in another batch. #3788 is a PR where I'm fixing thing / re-adding things back as I find them.

File: src/functions/template-tags/general.php [hunk 5]
```diff
@@ -5,2032 +5,1733 @@
...
 	 *
-	 * @category Events
+	 * This will remove all text that resembles a shortcode [shortcode 5].
 	 *
-	 * @param string $view Name of view to check, should match what's in Tribe__Events__Main->displaying when on that view
+	 * @since 5.1.0
 	 *
-	 * @return bool
-	 **/
-	function tribe_events_is_view_enabled( $view ) {
```
Comment (borkweb) [inline src/functions/template-tags/general.php]: I brought this back and [refactored it](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d405ece6fce146a7eb1fc2f822e4d10626724fb92519bdd94881a48d65684752R1313) to use Views\V2\Manager.

**Review by borkweb — COMMENTED** [review]
Wrapped up my review with more tweaks, re-additions, and deprecations!

File: src/Tribe/Main.php [hunk 6]
```diff
@@ -1572,64 +1489,6 @@ public function get_event_label_singular() {
...
-		 *
-		 * Disabling this behavior always is possible with:
-		 *
-		 *     add_filter( 'tribe_events_add_no_index_meta', '__return_false' );
-		 *
-		 *  Enabling it for all event views is possible with:
-		 *
-		 *     add_filter( 'tribe_events_add_no_index_meta', '__return_true' );
-		 */
-		public function issue_noindex() {
```
[the remark this answers is not in the document]
Comment (bordoni) [reply src/Tribe/Main.php:1599]: Going to bring this back I think I made a mistake here.

**Pull request closed without merging**